proto/gds/models/v1/models.proto

```diff
@@ -8,29 +8,36 @@ option go_package = "github.com/trisacrypto/directory/pkg/gds/models/v1;models";
 // issuance and only public keys and certificate metadata should be exchanged between
 // directories.
 message CertificateRequest {
-    // ID only needs to be unique with the requesting directory service.
+    // A unique identifier generated by the directory service, should be a globally
+    // unique identifier generated by the replica specified in requesting_replica.
```

Comment:
We should replicate certificate requests even though only one replica will handle the request. That way if the owner of the certificate request dies, it will be "automatically" backed up via anti-entropy and can then submit the certificate request. This means we need to track the owner of the certificate request, so I've added the "requesting replica" field below.

Reply:
Sounds good; glad we decided to name the requesting replica `owner` since that will make it align nicely with the `owner` field on `Object`